substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -64,19 +64,9 @@ extern crate alloc;
 use alloc::boxed::Box;
 use codec::{Decode, DecodeWithMemTracking, Encode, MaxEncodedLen};
 use core::{fmt::Debug, marker::PhantomData};
-use scale_info::TypeInfo;
-use sp_arithmetic::traits::{Saturating, Zero};
-use sp_runtime::RuntimeDebug;
-
-use frame_support::{
-	defensive,
-	dispatch::DispatchResultWithPostInfo,
-	ensure, impl_ensure_origin_with_arg_ignoring_arg,
-	traits::{
-		tokens::Balance as BalanceTrait, EnsureOrigin, EnsureOriginWithArg, Get, RankedMembers,
-		RankedMembersSwapHandler,
-	},
-	BoundedVec, CloneNoBound, EqNoBound, PartialEqNoBound, RuntimeDebugNoBound,
+use frame::{
+	prelude::*,
```

Comment:
```suggestion
	prelude::{*, Member as _},
```

Please try if this may fixes the issue with the ambiguous names.